#! /bin/sh /usr/share/dpatch/dpatch-run
## site.conf.dpatch by  <apollock@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Not exactly sure what this does

@DPATCH@
diff -urNad dhcp3-3.0.2/site.conf /tmp/dpep.E8xg8r/dhcp3-3.0.2/site.conf
--- dhcp3-3.0.2/site.conf	1999-07-08 01:20:10.000000000 +1000
+++ /tmp/dpep.E8xg8r/dhcp3-3.0.2/site.conf	2005-06-15 12:48:03.657178152 +1000
@@ -1,2 +1,3 @@
 # Put local site configuration stuff here to override the default
 # settings in Makefile.conf
+SCRIPT=debian
